### PR TITLE
Add test profitability

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/shared/components/profitability/base-profitability.tsx [hunk 1]
```diff
@@ -3,29 +3,32 @@ import * as React from "react";
...
 import { PROFITABILITY_VARIANT } from "./profitability.helper";
 
-interface IBaseProfitabilityProps {
+const BaseProfitability: React.FC<Props> = ({
+  className,
+  variant,
+  isPositive,
+  isNegative,
+  children
+}) => {
```
Comment (pavelzubov) [inline packages/shared/components/profitability/base-profitability.tsx]: Change to `=> (<div ...`

File: packages/shared/components/profitability/base-profitability.tsx [hunk 2]
```diff
@@ -3,29 +3,32 @@ import * as React from "react";
...
+        "profitability--negative": isNegative,
+        "profitability--chips": variant === PROFITABILITY_VARIANT.CHIPS
+      })}
+    >
+      {children}
+    </div>
+  );
+};
+
+export default React.memo(BaseProfitability);
```
Comment (pavelzubov) [inline packages/shared/components/profitability/base-profitability.tsx]: Change to `const BaseProfitability = React.memo(_BaseProfitability); export default BaseProfitability`

File: packages/shared/components/profitability/profitability.helper.ts [hunk 3]
```diff
@@ -9,25 +9,29 @@ export enum PROFITABILITY_VARIANT {
...
+  positive: `${String.fromCharCode(8593)} `
+};
+export const noPrefix = {
+  negative: "",
+  positive: ""
+};
+
 export const composeProfitabilityPrefix = (
   prefix: PROFITABILITY_PREFIX
 ): { negative: string; positive: string } => {
```
Comment (pavelzubov) [inline packages/shared/components/profitability/profitability.helper.ts]: You can create interface for `{ negative: string; positive: string }` and use it in out of `composeProfitabilityPrefix`, type of `signs`, `arrows`, `noPrefix` and other places

File: packages/shared/components/profitability/profitability.tsx [hunk 4]
```diff
@@ -9,42 +9,40 @@ import {
...
+    >
+      <>
+        {renderPrefix(value, prefix)}
+        {children}
+      </>
+    </BaseProfitability>
+  );
+};
+
+export default React.memo(Profitability);
```
Comment (pavelzubov) [inline packages/shared/components/profitability/profitability.tsx]: Change to `const Profitability = React.memo(_Profitability); export default Profitability `

File: packages/shared/components/profitability/profitability.tsx [hunk 5]
```diff
@@ -9,42 +9,40 @@ import {
   composeProfitabilityPrefix
 } from "./profitability.helper";
 
-interface IProfitabilityProps {
+const renderPrefix = (value: number | string, prefix: PROFITABILITY_PREFIX) => {
```
Comment (pavelzubov) [inline packages/shared/components/profitability/profitability.tsx]: Choose the exact type for `value` instead `number | string`

File: packages/shared/components/profitability/profitability.tsx [hunk 6]
```diff
@@ -9,42 +9,40 @@ import {
...
+      </>
+    </BaseProfitability>
+  );
+};
+
+export default React.memo(Profitability);
+
+interface Props {
   className?: string;
   value: number | string;
```
Comment (pavelzubov) [inline packages/shared/components/profitability/profitability.tsx]: Choose the exact type for `value` instead `number | string`

**Comment by Anchello** [comment]
Fixed all comments

**Pull request merged**